src/qt/bitcoin.cpp

```diff
@@ -556,6 +556,9 @@ int GuiMain(int argc, char* argv[])
     /// 9. Main GUI initialization
     // Install global event filter that makes sure that long tooltips can be word-wrapped
     app.installEventFilter(new GUIUtil::ToolTipToRichTextFilter(TOOLTIP_WRAP_THRESHOLD, &app));
+    // Install global event filter that makes sure that out-of-focus labels do not contain text cursor.
+    // This is a temporary fix of QTBUG-59514.
```

Comment:
nit, move/copy this comment to class doc.

Reply:
[Updated](https://github.com/bitcoin/bitcoin/pull/19210#issuecomment-653598259).